src/current/_includes/v24.1/essential-metrics.md

```diff
@@ -149,7 +149,7 @@ The **Usage** column explains why each metric is important to visualize in a cus
 | jobs.backup.currently_running                         | {% if include.deployment == 'self-hosted' %}jobs.backup.currently_running |{% elsif include.deployment == 'dedicated' %}NOT AVAILABLE |{% endif %} Number of backup jobs currently running                      | See Description.                                             |
 | jobs.backup.currently_paused                          | {% if include.deployment == 'self-hosted' %}jobs.backup.currently_paused |{% elsif include.deployment == 'dedicated' %}NOT AVAILABLE |{% endif %} Number of backup jobs currently considered Paused            | Monitor and alert on this metric to safeguard against an inadvertent operational error of leaving a backup job in a paused state for an extended period of time. In functional areas, a paused job can hold resources or have concurrency impact or some other negative consequence. Paused backup may break the [recovery point objective (RPO)]({% link {{ page.version.version }}/backup.md %}#performance). |
 | schedules.BACKUP.failed                               | {% if include.deployment == 'self-hosted' %}schedules.backup.failed |{% elsif include.deployment == 'dedicated' %}NOT AVAILABLE |{% endif %} Number of BACKUP jobs failed                                 | Monitor this metric and investigate backup job failures.                  |
-| schedules.BACKUP.last-completed-time                  | {% if include.deployment == 'self-hosted' %}schedules.backup.last_completed_time |{% elsif include.deployment == 'dedicated' %}NOT AVAILABLE |{% endif %} The unix timestamp of the most recently completed backup by a schedule specified as maintaining this metric | Monitor this metric to know that backups are meeting the [recovery point objective (RPO)]({% link {{ page.version.version }}/backup.md %}#performance). |
+| schedules.BACKUP.last-completed-time                  | {% if include.deployment == 'self-hosted' %}schedules.backup.last_completed_time |{% elsif include.deployment == 'dedicated' %}NOT AVAILABLE |{% endif %} The unix timestamp of the most recently completed backup by a schedule specified as maintaining this metric | Monitor this metric to know that backups are meeting the [recovery point objective (RPO)]({% link {{ page.version.version }}/backup.md %}#performance). Each node exports the time that it last completed a backup on behalf of the schedule. If a node is restarted, it will report `0` until it completes a backup. If all nodes are restarted, you would see the `max()` be zero until a node completes a backup.<br><br> To successfully use this metric, first, from each node take the maximum over a rolling window equal to or greater than the backup frequency, and then take the maximum of those values across nodes. For example with a backup frequency of 60 minutes, monitor `time() - max_across_nodes(max_over_time(schedules_BACKUP_last_completed_time, 60min))`. |
```

Comment:
Made some suggestions here which may not be highlighted by GitHub.

One thing I noticed but didn't edit:

The backup.md#performance section may not be the best place to link for RPO, because it offers a similar use of RPO ('this is important for RPO'); I think it would be better to link to https://www.cockroachlabs.com/docs/stable/disaster-recovery-overview#resilience-strategy, which has a definition of RPO, and then find a place to add on that page a mention of the monitoring that you're explaining, linking back to Essential Metrics.

Feel free to hold off on any/all of these changes ... or perhaps discuss with Kathryn or create a follow up issue.